src/pages/ContactPage.jsx

```diff
@@ -1,7 +1,12 @@
 import React from 'react';
+import ContactContainer from '../containers/Contact/ContactContainer';
```

Comment:
ini bisa dipanggil seperti ini
```suggestion
import ContactContainer from 'containers/Contact/ContactContainer';
```